stempy/image.cpp

```diff
@@ -386,11 +395,76 @@ void radialSumFrame(int centerX, int centerY, const uint16_t data[],
   }
 }
 
+namespace {
+
+#ifdef VTKm
+struct RadialSumWorklet : public vtkm::worklet::WorkletMapField
+{
+  vtkm::Vec<int, 2> m_center;
+  vtkm::Vec<int, 2> m_shape;
+  int m_imageNumber;
+  uint32_t m_numberOfScanPositions;
+
+  RadialSumWorklet(vtkm::Vec<int, 2> center, vtkm::Vec<int, 2> shape,
+                   int imageNumber, uint32_t numberOfScanPositions)
+    : m_center(center), m_shape(shape), m_imageNumber(imageNumber),
+      m_numberOfScanPositions(numberOfScanPositions){};
+
+  using ControlSignature = void(FieldIn, AtomicArrayInOut);
+
+  using ExecutionSignature = void(_1, _2, WorkIndex);
+
+  template <typename AtomicInterface>
+  VTKM_EXEC void operator()(const vtkm::UInt16& value,
+                            AtomicInterface& radialSum, vtkm::Id i) const
+  {
+    auto x = i % m_shape[0];
+    auto y = i / m_shape[0];
```

Comment:
Is `m_shape[1]` supposed to be used anywhere? I haven't seen it being used.

Reply:
Nope, I guess I can just pass in the width

Reply:
Done